api_tests/preprints/filters/test_filters.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+import pytest
+from mock import MagicMock
+
+class PreprintsListFilteringMixin(object):
+
+    def setUp(self):
+        super(PreprintsListFilteringMixin, self).setUp()
+
+        self._setUp()
```

Comment:
See https://github.com/CenterForOpenScience/osf.io/pull/6940/files#r106475339 

Not a blocker, but could be refactored.